### PR TITLE
Basic CRUD TC-s

### PR DESCRIPTION
**Pull request by PavloPetrina**
Removed email from createOrganizationRequest
Applied google code format
Refactor tests a bit (created the separated `Base` class for them)

### CODE REVIEW

File: scalecube-organization/src/test/java/resources/features/mpa-7229-Configuration-service-auto-tests-part-2-Organization-service.feature [hunk 1]
```diff
@@ -0,0 +1,822 @@
...
+    And only single organization "organizationId" with specified "name" and "email" already created and owned by user "A"
+    When the user "A" requested to get the own organization info
+    Then user "A" should receive the successful response with relevant organization data
+
+
+  #MPA-7229 (#2.1) - SHOULD WE ALLOW IF THE ORIGIN OWNER WAS REMOVED ???
+  Scenario: Successful get of specific Organization info upon the origin owner was removed from own Organization
+    Given the user "A" have got a valid "token" issued by relevant authority
+    And only single organization "organizationId" with specified "name" and "email" already created and owned by user "A"
+    And the user "A" was removed from user's "A" organization by its own decision
```
Comment (aharonha) [inline scalecube-organization/src/test/java/resources/features/mpa-7229-Configuration-service-auto-tests-part-2-Organization-service.feature]: @PavloPetrina an owner can not remove himself. there is a need of another one

Comment (PavloPetrina) [reply scalecube-organization/src/test/java/resources/features/mpa-7229-Configuration-service-auto-tests-part-2-Organization-service.feature]: Is it a business requirement? If so this is negative TC

**Comment by PavloPetrina** [comment]
Granulated for 3 pull requests

**Comment by aharonha** [comment]
@PavloPetrina we have 2 test cases that are not covered in this PR:  
* `OrganizationServiceApiKeyTest`
* `DeleteOrganizationApiKeyTest`
let's merge this

**Comment by PavloPetrina** [comment]
> @PavloPetrina we have 2 test cases that are not covered in this PR:
> 
> * `OrganizationServiceApiKeyTest`
> * `DeleteOrganizationApiKeyTest`
>   let's merge this

Before the Merge pls eventually, remove the "email" validation from the "/create" request > http://take.ms/1XVja as it still required (locally deployed Docker with your branch skipped the tests) but actually tests are falling pls take a look > http://take.ms/xwUodO

**Comment by aharonha** [comment]
but they are the scenarios, this one also has some development

**Comment by PavloPetrina** [comment]
> but they are the scenarios, this one also has some development

So what is the proposal > merge anyway? Skip the Codacy and API refactoring regarding the email cut off?

**Pull request merged**